lib/core/src/display/shape/primitive/shader/builder.rs

```diff
@@ -0,0 +1,64 @@
+//! This module contains GLSL code builder. It allows translating complex vector shapes to the GLSL
+//! code.
+
+use crate::prelude::*;
+
+use super::canvas::Canvas;
+use super::super::class::Shape;
+use crate::display::shape::primitive::def::sdf;
+use crate::display::symbol::shader::builder::CodeTemplete;
+use crate::display::shape::primitive::shader::overload;
+
+
+// ===============
+// === Builder ===
+// ===============
+
+// === GLSL Sources ===
+
+const MATH            :&str = include_str!("../glsl/math.glsl");
+const COLOR           :&str = include_str!("../glsl/color.glsl");
+const DEBUG           :&str = include_str!("../glsl/debug.glsl");
+const SHAPE           :&str = include_str!("../glsl/shape.glsl");
+const FRAGMENT_RUNNER :&str = include_str!("../glsl/fragment_runner.glsl");
+
+
+// === Definition ===
+
+/// GLSL code builder.
+pub struct Builder {}
```

Comment:
I was thinking about it. Let me keep it this way for now. I will revisit this design in my next PR. I will add note there, but I feel it will contain fields soon.